proto/storage/alert.proto

```diff
@@ -114,6 +114,12 @@ message Alert {
   string namespace = 20; // @gotags: search:"Namespace,store"
   string namespace_id = 21; // @gotags: search:"Namespace ID" sql:"type(uuid)"
 
+  enum EntityType {
```

Comment:
Should we have an `unset` or `unknown`?

Reply:
I don't think there can be an alert with no entity, but it makes sense to have UNSET as the default enum value.